packages/beacon-node/src/util/blobs.ts

```diff
@@ -74,7 +74,7 @@ export function computeBlobSidecars(
 export function computeDataColumnSidecars(
   config: ChainForkConfig,
   signedBlock: SignedBeaconBlock,
-  contents: deneb.Contents & {kzgCommitmentsInclusionProof?: fulu.KzgCommitmentsInclusionProof}
+  contents: fulu.Contents & {kzgCommitmentsInclusionProof?: fulu.KzgCommitmentsInclusionProof}
```

Comment:
make the contents forkaware and have the fulu ones store the cells computed once in produceblockbody...

Reply:
Is there any concern that adding the cells to block contents will differ from the [spec](https://github.com/ethereum/beacon-APIs/blob/2b1d7b5ac4756881bd29e7adacc9b7032343d981/types/fulu/block_contents.yaml#L9)?